python/kserve/kserve/model_server.py

```diff
@@ -172,7 +176,7 @@ async def start(self, models: Union[List[Model], Dict[str, Deployment]]) -> None
         )
 
         self._server = uvicorn.Server(cfg)
-        servers = [self._server.serve()]
+        servers = [self._server.serve(), self._grpc_server.start(self.workers)]
```

Comment:
@yuzisun @sukumargaonkar and I figured out a way to make the `start()` function *sync again :D 

```
    def start()
      ...

        async def servers_task():
            servers = [self._server.serve(), dummyserver.serve()]
            await asyncio.gather(*servers)

        asyncio.run(servers_task())
```

so we can get rid of all the `asyncio.run()` in the other model servers